### DISCUSSION TITLE
Retrieve all existing object identifiers.

### DISCUSSION THREAD

**Discussion by igr** [vexy/Fridge#52] [General]
This might be useful during

+ development - want to clean up my mess :)
+ when migrating data - get existing structures and convert them to new ones.
+ data structure validation - when data is important

**Comment by vexy** [comment]
Something similar is already cooking:

https://github.com/vexy/Fridge/issues/24

Although, I understand the needs, it would throw even more responsibility into a layer that's technically just a thin generically coded wrapper.

Fridge is meant to be just stupid fetch and store mechanism.

The rest is "upper layer" logic.

With `Identifiable` in place, methods for IO will get another overloaded function to handle the `struct`.

But, nice suggestions to think about.

**Comment by vexy** [comment]
Here's another issue created out of this, so that we can expand it if needed: https://github.com/vexy/Fridge/issues/46

**Comment by igr** [comment]
Yeah, agree! I totally understand, less complexity is better, so far I like it

**Comment by vexy** [comment]
But this part ☝️

> development - want to clean up my mess..

Hmm 🤔 I understand this need quite well. Yup it can get messy over time.

Right now there is `drop(..)` method. Pass the id and the object (file) is physically erased.

Perhaps expanding it to `dropAll` ? 🤔
